runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -134,6 +134,10 @@ Example: >
   programs can set this by emitting an escape sequence.
 - |'channel'|  Terminal PTY |job-id|.  Can be used with |chansend()| to send
   input to the terminal.
+- The |TermClose| event gives the terminal job exit code in the |v:event|
+  "status" field. For example, this autocmd closes terminal buffers if the job
+  exited without error: >
+    autocmd TermClose * if !v:event.status | exe 'bdelete! '..expand('<abuf>') | endif
```

Comment:
This should be just one `.` for concatenation, not two. And in fact you don’t need to concatenate the strings at all since `exe` automatically separates its arguments with a space.

Reply:
> This should be just one `.` for concatenation, not two. And in fact you don’t need to concatenate the strings at all since `exe` automatically separates its arguments with a space.

Two is supported now and is less ambiguous. `:help expr-..`

I thought about omitting the operator but felt like it's a bit too magical and could lead to mistakes if users edit this in their config without knowing all the fun details of vimscript.

Reply:
> I’m still surprised this is necessary. Using just `bdelete` works fine for me even with multiple terminal buffers open

Try `:terminal sleep 3` then switch to another buffer. 

Neither terminal_close() nor apply_autocmds_group() switch the current buffer, afair.

Reply:
> Two is supported now and is less ambiguous. `:help expr-..`

Ah that’s right, I had forgotten about that since I so rarely see it in practice. That is nice that the Lua and VimL concatenation operator can be the same.